src/main/java/seedu/linkedout/logic/commands/DeleteCommand.java

```diff
@@ -8,21 +8,21 @@
 import seedu.linkedout.commons.core.index.Index;
 import seedu.linkedout.logic.commands.exceptions.CommandException;
 import seedu.linkedout.model.Model;
-import seedu.linkedout.model.person.Person;
+import seedu.linkedout.model.applicant.Applicant;
 
 /**
- * Deletes a person identified using it's displayed index from the linkedout book.
```

Comment:
Line 14:
Deletes a applicant -> Deletes an applicant
Linkedout book -> Linkedout

Can consider reviewing the comments on this line.

Reply:
Thanks for the spot! Updated.